src/messages/nl.json

```diff
@@ -12,8 +12,13 @@
     "sortNameAsc": "Naam - Oplopend",
     "sortNameDesc": "Naam - Aflopend",
     "fetchError": "Er is een probleem opgetreden bij het ophalen van de datasets. Laad de pagina opnieuw om het opnieuw te proberen.",
-    "filters": "filters",
-    "clearAllFilters": "Alles verwijderen"
+    "filters": "Filters",
+    "clearAllFilters": "Alles verwijderen",
+    "aria": {
```

Comment:
Is it necessary to use a separate object for this (i.e. do we want to make clear in the locale files that these are aria-related translations)?

Reply:
I thought this was relevant. Because, in this case, I do think the context is important. For screen readers, you want more descriptive text.

Reply:
I have flattered these keys. And prefixed the keys with 'accessibility'.